src/components/AnnotationBrowser/AnnotationList.vue

```diff
@@ -246,6 +275,24 @@ export default class AnnotationList extends Vue {
     );
   }
 
+  get selectAllIndeterminate() {
+    const nSelected = this.selectedItems.length;
+    return nSelected > 0 && nSelected < this.filteredItems.length;
+  }
+
+  get selectAllValue() {
+    const nSelected = this.selectedItems.length;
+    return nSelected === this.filteredItems.length;
```

Comment:
Not sure `nSelected` is needed here. I think I'll go with a simple: `this.selectedItems.length === this.filteredItems.length;`